ios/FluentUI/Navigation/Shy Header/ShyHeaderController.swift

```diff
@@ -126,7 +125,11 @@ class ShyHeaderController: UIViewController {
 
         updatePadding()
         setupNotificationObservers()
-        updateNavigationBarStyle()
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
```

Comment:
Isn't viewDidAppear too late for doing appearance changes? As in, won't there be a blue flicker if we do the update at this point?

Reply:
Updating in viewWillAppear doesn't always reliably access the right theme, so updating colors in viewDidAppear is safer (which was probably why it was there beforehand). Also, I wasn't able to reproduce the issue you mentioned on this branch since the NavigationController handles the color updates with the correct theme from now on.